dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
 trace.advanced=Use the advanced trace format.
 
 isoDateFormat=Use the ISO 8601 date format
-isoDateFormat.desc=If this attribute is set to true, log entries in the messages.log, trace.log, and FFDC log files will use the ISO 8601 date and time format. The default value is false.
+isoDateFormat.desc=The date and time use a locale-specific format or the ISO-8601 format. You can specify true or false for the value of the attribute or the value of the equivalent property. The default value is false. Avoid trouble: Use the com.ibm.ws.logging.isoDateFormat property, except in cases where you are trying to change the configuration dynamically after startup. \
+If you specify a value of true, the ISO-8601 format is used in the messages.log file, the trace.log file, and the FFDC logs. The format is yyyy-MM-dd'T'HH:mm:ss.SSSZ. \
+If you specify a value of false, the date and time are formatted according to the default locale set in the system. If the default locale is not found, the format is dd/MMM/yyyy HH:mm:ss:SSS z.
 
 filter.sensitive=Suppress sensitive trace
 filter.sensitive.desc=The server trace can expose sensitive data when tracing untyped data, such as bytes received over a network connection. \
-If true, prevent potentially sensitive information from being exposed in log and trace files. The default value is false.
+If true, prevent potentially sensitive information from being exposed in log and trace files. The default value is false. Avoid trouble: Use the com.ibm.ws.logging.filter.sensitive property, except in cases where you are trying to change the configuration dynamically after startup.
 
 hide.messages=Messages to be hidden
-hide.messages.desc=The list of messages, separated by a comma, that are configured to be hidden from the console.log and message.log files. If the messages are configured to be hidden, then they are redirected to the trace.log file.
+hide.messages.desc=The list of messages, separated by a comma, that are configured to be hidden from the console.log and message.log files. If the messages are configured to be hidden, then they are redirected to the trace.log file. Avoid trouble: Use the com.ibm.ws.logging.hide.messages property, except in cases where you are trying to change the configuration dynamically after startup.
 
 message.source=Message source
-message.source.desc=The list of sources to route to messages.log
+message.source.desc=The list of comma-separated sources that route to the messages.log file. This property only applies when messageFormat=json. Valid values are message, trace, accessLog, ffdc, and audit. By default, messageSource is set to the environment variable WLP_LOGGING_MESSAGE_SOURCE (if set), or message. Note: To use the audit source, enable the Liberty audit feature. Avoid trouble: Use the WLP_LOGGING_MESSAGE_SOURCE environment variable or com.ibm.ws.logging.message.source property, except in cases where you are trying to change the configuration dynamically after startup.
```

Comment:
Change
"This property only applies when messageFormat=json."
to
"This property applies only when messageFormat=json."
and change
"Avoid trouble: Use the WLP_LOGGING_MESSAGE_SOURCE environment variable or com.ibm.ws.logging.message.source property, except in cases where you are trying to change the configuration dynamically after startup."
to
"Avoid trouble: Use the WLP_LOGGING_MESSAGE_SOURCE environment variable or the com.ibm.ws.logging.message.source property, except in cases where you are trying to change the configuration dynamically after startup."